doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+
+
+### 2D Images ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut/gesture
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame image                           :kbd:`F`
+Center image at 1:1 scale             :kbd:`Home`
+Toggle red channel                    :kbd:`R`
+Toggle green channel                  :kbd:`G`
+Toggle blue channel                   :kbd:`B`
+Toggle alpha channel                  :kbd:`A`
+===================================== =============================================
+```
+
+### 3D Scenes ###
+
+```eval_rst
+========================================== =====================================
+Action			                           Shortcut/gesture
+========================================== =====================================
+Tumble                                     :kbd:`Alt`-click and drag
+Track                                      :kbd:`Alt`-middle-click and drag
+Dolly                                      :kbd:`Alt`-right-click and drag
+"                                          Mouse wheel up/down
+Expand selection                           :kbd:`↓`
+Fully expand selection                     :kbd:`Shift`-:kbd:`↓`
+Collapse selection                         :kbd:`↑`
+Frame selection                            :kbd:`F`
+Fit clipping planes to selection           :kbd:`Ctrl`-:kbd:`K`
+Frame and fit clipping planes to selection :kbd:`Ctrl`-:kbd:`F`
+========================================== =====================================
+```
+
+
+## _Script Editor_ Tab ##
+
+### Text Entry ###
+
+```eval_rst
+===================================== =============================================
+Action                                Shortcut/gesture
+===================================== =============================================
+Pull node into script                 Middle-click and drag node in *Node Graph* tab
+                                      onto *Script Editor* tab's text input field
+Pull plug into script                 Middle-click and drag plug in *Node Graph* 
+                                      tab onto *Script Editor* tab's text input field
+"                                     Click and drag plug label in *Node Editor*
+                                      tab onto *Script Editor* tab's text input field
+Pull plug value into script           :kbd:`Shift`-click and drag plug label in
+                                      *Node Editor* tab onto *Script Editor* tab's
+                                      text input field
+Pull color value into script          Click and drag a spot in *Viewer* tab onto
```

Comment:
This should mention it needs to be a 2d viewer. Also it works from color widgets, not just viewers.